### ISSUE TITLE
DependencyCheck 8.0.0 Maven output cannot be processed in SonarQube plugin

### ISSUE THREAD

**Issue by mind-overflow** [dependency-check/DependencyCheck#5310]
**Describe the bug**
I generate the dependency check reports with:
```
mvn dependency-check:check
```
The reports get generated correctly:
```
[INFO] Analysis Complete (4 seconds)
[INFO] Writing report to: /drone/src/target/dependency-check-report.json
[INFO] Writing report to: /drone/src/target/dependency-check-report.html
[WARNING] 

One or more dependencies were identified with known vulnerabilities in <ProjectName>:

kotlin-stdlib-1.4.10.jar (pkg:maven/org.jetbrains.kotlin/kotlin-stdlib@1.4.10, cpe:2.3:a:jetbrains:kotlin:1.4.10:*:*:*:*:*:*:*) : CVE-2020-29582, CVE-2022-24329
kotlin-stdlib-common-1.4.0.jar (pkg:maven/org.jetbrains.kotlin/kotlin-stdlib-common@1.4.0, cpe:2.3:a:jetbrains:kotlin:1.4.0:*:*:*:*:*:*:*) : CVE-2020-15824, CVE-2020-29582, CVE-2022-24329
snakeyaml-1.33.jar (pkg:maven/org.yaml/snakeyaml@1.33, cpe:2.3:a:snakeyaml_project:snakeyaml:1.33:*:*:*:*:*:*:*) : CVE-2022-1471

See the dependency-check report for more details.

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:29 min
[INFO] Finished at: 2023-01-16T04:27:39Z
[INFO] ------------------------------------------------------------------------
```
I then proceed to run the full SonarQube scan with:
```
mvn sonar:sonar
```

However, the dependency check sensor part seems to fail without any output:
```
[INFO] ------------- Run sensors on project
[INFO] Sensor Dependency-Check [dependencycheck]
[INFO] Process Dependency-Check report
[INFO] Using JSON-Reportparser
[WARNING] JSON-Analysis aborted
[INFO] Using XML-Reportparser
[INFO] Dependency-Check XML report does not exists. Please check property sonar.dependencyCheck.xmlReportPath:/drone/src/${WORKSPACE}/dependency-check-report.xml
[INFO] XML-Analysis skipped/aborted due to missing report file
[INFO] Upload Dependency-Check HTML-Report
[INFO] Process Dependency-Check report (done) | time=220ms
[INFO] Sensor Dependency-Check [dependencycheck] (done) | time=224ms
```

**Version of dependency-check used**
The problem occurs using version 8.0.0 of the Maven plugin.
I'm running the latest Sonar plugin version available from the Marketplace (v3.0.1) on SonarQube 9.8.0.63668.

**Additional context**
Reverting to the Maven plugin version 7.4.4 solves the issue and the JSON report is handled correctly:
```
[INFO] ------------- Run sensors on project
[INFO] Sensor Dependency-Check [dependencycheck]
[INFO] Process Dependency-Check report
[INFO] Using JSON-Reportparser
[INFO] Linking 33 dependencies
[INFO] Upload Dependency-Check HTML-Report
[INFO] Process Dependency-Check report (done) | time=286ms
[INFO] Sensor Dependency-Check [dependencycheck] (done) | time=286ms
```

Concerned Maven `pom.xml` configuration:
```xml
    <properties>
        <maven.compiler.source>16</maven.compiler.source>
        <maven.compiler.target>16</maven.compiler.target>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <sonar.dependencyCheck.htmlReportPath>./target/dependency-check-report.html</sonar.dependencyCheck.htmlReportPath>
        <sonar.dependencyCheck.jsonReportPath>./target/dependency-check-report.json</sonar.dependencyCheck.jsonReportPath>
        <sonar.dependencyCheck.summarize>true</sonar.dependencyCheck.summarize>
    </properties>

<...>
    <build>
        <...>
        <plugins>
            <...>
            <plugin>
                <groupId>org.sonarsource.scanner.maven</groupId>
                <artifactId>sonar-maven-plugin</artifactId>
                <version>3.9.1.2184</version>
            </plugin>

            <plugin>
                <groupId>org.owasp</groupId>
                <artifactId>dependency-check-maven</artifactId>
                <version>8.0.0</version>
                <configuration>
                    <formats>
                        <format>html</format>
                        <format>json</format>
                    </formats>
                </configuration>
            </plugin>
        </plugins>
    </build>

```

**Comment by aikebah** [comment]
Looks like the Sonar plugin needs an update for the updated json report

**Comment by aikebah** [comment]
See also https://github.com/dependency-check/dependency-check-sonar-plugin/issues/748

**Comment by mind-overflow** [comment]
> See also [dependency-check/dependency-check-sonar-plugin#748](https://github.com/dependency-check/dependency-check-sonar-plugin/issues/748)

Yes! Looks like the same issue - just in a different repo. Thanks for the quick reply! I guess this one can be closed if it doesn't come from DependencyCheck itself but from the SonarQube plugin.

**Comment by evyaroshevich** [comment]
Hi, is there any news on solving this problem?

**Comment by jeremylong** [comment]
No clue - I'd check https://github.com/dependency-check/dependency-check-sonar-plugin/issues/748

The maintainers of dependency-check do not also maintain the sonar-plugin.

**Comment by aikebah** [comment]
@evyaroshevich It's released by them in v3.1.0

https://github.com/dependency-check/dependency-check-sonar-plugin/releases/tag/3.1.0

**Comment by jnt2007** [comment]
Had same problem
```
[WARNING] JSON-Analysis aborted
```
Update to 3.1.0 version of Sonar plugin helped. Thanks.
